### PR TITLE
if we close these, we can all go home

### PR DESCRIPTION
**Pull request by manavortex**
# fix code analyser findings

### CODE REVIEW

File: WolvenKit.App/ViewModels/Dialogs/SoundModdingViewModel.cs [hunk 1]
```diff
@@ -79,26 +79,28 @@ private void LoadInfo()
         }
 
         var path = Path.Combine(Environment.CurrentDirectory, "Resources", "soundEvents.json");
-        path = Path.Combine(_projectManager.ActiveProject.ResourcesDirectory, "info.json");
```
Comment (seberoth) [inline WolvenKit.App/ViewModels/Dialogs/SoundModdingViewModel.cs:82]: Now it reads the wrong file. Should be info.json, not soundEvents.json

File: WolvenKit.Modkit/RED4/Tasks/UncookTask.cs [hunk 2]
```diff
@@ -38,19 +38,20 @@ public int UncookTask(FileSystemInfo[] paths, UncookTaskOptions options)
...
-        }
 
         if (options.meshExportType != null && string.IsNullOrEmpty(options.meshExportMaterialRepo) && options.outpath is null)
         {
             _loggerService.Error("When using --mesh-export-type, the --outpath or the --mesh-export-material-repo must be specified.");
             return ERROR_INVALID_COMMAND_LINE;
         }
 
-        if (options.gamepath != null && options.gamepath.Exists)
+        if (options.outpath == null)
```
Comment (seberoth) [inline WolvenKit.Modkit/RED4/Tasks/UncookTask.cs:48]: Same "problem" as before. The analyzer would still complain that there is a double null check (L42)

File: WolvenKit.RED4/Types/Classes/ButtonCursorStateView.cs [hunk 3]
```diff
@@ -31,7 +31,7 @@ public CName DefaultStateName
 		public ButtonCursorStateView()
 		{
 			HoverStateName = "Hover";
-			PressStateName = "Hover";
+			PressStateName = "Press";
```
Comment (seberoth) [inline WolvenKit.RED4/Types/Classes/ButtonCursorStateView.cs:34]: Don't edit `WolvenKit.RED4.Types.Classes`. These are autogenerated.
If its wrong, its wrong at CDPRs end.

File: WolvenKit.App/ViewModels/Dialogs/SoundModdingViewModel.cs (hunk 1 above)
Comment (manavortex) [reply WolvenKit.App/ViewModels/Dialogs/SoundModdingViewModel.cs:82]: Oh, alright! I was guessing from context, but you actually know. Fixing :)

**Pull request merged**